### PR TITLE
Enable Previous Games View

### LINKED ISSUE #23: Game Archives
This is kind of implemented but is super glitchy and looks bad. It's technically on the public site, but not linked cuz it doesn't look nice. Can we fix this?

### PR DESCRIPTION
**Pull request by pkrshnan**
Changes:
- Enabled previous games button on nav bar for landing pages
- Fixed broken collapses on missions page
- Redesigned entire previous games page

Fixes #23

### CODE REVIEW

File: app/templates/jinja2/previous_games.html [hunk 1]
```diff
@@ -31,32 +31,32 @@
...
-<h2>{{ game_rendering }} Game Zombie Tree</h2>
-<div id="zombie-tree"></div>
-</section>
-<br><br>
-</main>
+  <section class="game-summary" id="intro">
+    <h1 class="prev-games-title">{{ game_rendering }} Game Zombie Tree</h1>
+    <div id="zombie-tree"></div>
+    <footer>
+      <p>&copy; UWaterloo Humans vs Zombies Society <time>2017</time> &ndash; <time><script>document.write(new Date().getFullYear())</script></time>.</p>
```
Comment (tig567899) [inline app/templates/jinja2/previous_games.html:56]: Why is this hard coded?

File: app/templates/jinja2/missions.html [hunk 2]
```diff
@@ -32,7 +32,7 @@
 </header>
 
 <main class="ui-layout" id="landing">
-  <div class="mission_details">
+  <div class="full-height">
```
Comment (tig567899) [inline app/templates/jinja2/missions.html:35]: Why?

Comment (pkrshnan) [reply app/templates/jinja2/missions.html:35]: I made a reusable class that essentially we can use in a bunch of places that will make the page full height.

File: app/templates/jinja2/previous_games.html (hunk 1 above)
Comment (pkrshnan) [reply app/templates/jinja2/previous_games.html:56]: It's hardcoded in a bunch of places, same with the nav. As part of the next PR which is to redo the mobile website, I plan to move this all to reusable components.

**Review by tig567899 — APPROVED** [review]

**Pull request merged**